src/index.tsx

```diff
@@ -18,4 +25,12 @@ app.get("/", (c) => {
 	);
 });
 
+app.get("/board", async (c) => {
+	const vars = env<Env>(c);
```

Comment:
tested locally, I had to change it to this get it to work
```suggestion
	const vars = env<Env>(c, "workerd");
```
see: [Hono Documentation - Adapter Helper - Specify the runtime](https://hono.dev/docs/helpers/adapter#specify-the-runtime)